EventGenerator/src/CRYEventGenerator_module.cc

```diff
@@ -72,6 +76,20 @@ namespace mu2e {
     mf::LogInfo("CRYEventGenerator") << oss.str();
   }
 
+  void CryEventGenerator::endSubRun(art::SubRun &subrun)
+  {
+    // All inputs (except getLiveTime) in CosmicLivetime are not used in the livetime calculation
+    // Livetime is calculated internally by cry: cryGen->getLiveTime()
+    std::unique_ptr<CosmicLivetime> livetime(new CosmicLivetime(1,
+                                                                cryGen->getSubboxLength(),
```

Comment:
I am confused by the units: this accessor claims to be a length, but the constructor needs an area.  Is the accessor incorrectly named, or should this value be squared?